server/elo.ts

```diff
@@ -0,0 +1,27 @@
+export function calculateElo(previousUserElo: number, score: number, foeElo: number): number {
+	// The K factor determines how much your Elo changes when you win or
+	// lose games. Larger K means more change.
+	// In the "original" Elo, K is constant, but it's common for K to
+	// get smaller as your rating goes up
+	let K = 50;
+
+	// dynamic K-scaling (optional)
+	if (previousUserElo < 1200) {
```

Comment:
Just noticed that these numbers differ from the loginserver (see https://github.com/smogon/pokemon-showdown-client/blob/master/lib/ntbb-ladder.lib.php#L395). Do `ladders-local` and `ladders-remote` need different dynamic K-scaling?

Reply:
Per Zarel in Discord, they're intentionally different due to the high activity of OU/randbats on the main server. I'll adjust this PR.

Reply:
Separated out, main server calc now matches the loginserver numbers with 2c8d3c5.